idl/forward_request.idl.hh

```diff
@@ -1,9 +1,23 @@
+namespace cql3 {
+namespace functions {
+class function_name {
+    sstring keyspace;
+    sstring name;
+};
+}
+}
 namespace query {
 struct forward_request {
+    struct aggregation_info {
+        cql3::functions::function_name name;
+        std::vector<sstring> column_names;
+    };
     enum class reduction_type : uint8_t {
         count,
+        aggregate
     };
-    std::vector<query::forward_request::reduction_type> reduction_types;
+
+    std::vector<std::pair<query::forward_request::reduction_type, std::optional<query::forward_request::aggregation_info>>> reductions;
```

Comment:
Won't this break compatibility with older versions?

Reply:
fixed